network-api/networkapi/wagtailpages/tests/buyersguide/test_utils.py

```diff
@@ -0,0 +1,46 @@
+from networkapi.wagtailpages.factory import buyersguide as buyersguide_factories
+from networkapi.wagtailpages.pagemodels.buyersguide.utils import get_bg_featured_cta
+from networkapi.wagtailpages.tests import base as test_base
+
+
+class TestGetFeaturedCallToActionFunction(test_base.WagtailpagesTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.pni_homepage = buyersguide_factories.BuyersGuidePageFactory(
+            parent=cls.homepage,
+        )
+        cls.content_index = buyersguide_factories.BuyersGuideEditorialContentIndexPageFactory(
+            parent=cls.pni_homepage,
+        )
+        cls.product_page = buyersguide_factories.ProductPageFactory(
+            parent=cls.pni_homepage,
+        )
+
+    def test_get_bg_featured_cta_with_product_page(self):
+        self.pni_homepage.call_to_action = buyersguide_factories.BuyersGuideCallToActionFactory()
+        self.pni_homepage.save()
+
+        featured_cta = get_bg_featured_cta(self.product_page)
+        self.assertIsNotNone(featured_cta)
```

Comment:
Can you add some context as to why are not checking directly the value of the featured CTA instead of checking that it is not `None`?

Reply:
Hey Tibor, no reason directly, however I do like this suggestion alot as it provides another layer of testing. Especially after you mentioned that the presence of _anything_ will allow assertIsNotNone to pass. Updated!